lib/livebook/hubs.ex

```diff
@@ -219,9 +231,9 @@ defmodule Livebook.Hubs do
   @doc """
   Updates a secret for given hub.
   """
-  @spec update_secret(Secret.t()) :: :ok | {:error, list({atom(), list(String.t())})}
-  def update_secret(%Secret{origin: {:hub, id}} = secret) do
-    {:ok, hub} = get_hub(id)
+  @spec update_secret(Provider.t(), Secret.t()) ::
+          :ok | {:error, list({atom(), list(String.t())})}
+  def update_secret(hub, %Secret{} = secret) do
     true = capability?(hub, [:update_secret])
```

Comment:
Last time we discussed we would remove `:update_secret` and `:delete_secret` capabilities and instead we will check that `false = secret.read_only`. And we will change all enterprise secrets to have `read_only: true`. Do we plan to do it now or in a separate PR?

Reply:
Nice catch, I forgot to update these.